src/internal/m365/onedrive/restore.go

```diff
@@ -779,12 +819,13 @@ func restoreData(
 
 	var (
 		item         = newItem(name, false)
+		isFolder     = item.GetFolder() != nil
```

Comment:
is this variable needed? The line above makes it seem like this is only restoring items because we only pass `false` to `newItem`